build.sbt

```diff
@@ -27,19 +25,11 @@ addCommandAlias(
   "; headerCreateAll; fixImports; scalafmtAll; fixCSS"
 )
 
-inThisBuild(
-  Seq(
-    homepage                      := Some(url("https://github.com/gemini-hlsw/explore")),
-    addCompilerPlugin(
-      ("org.typelevel"             % "kind-projector" % kindProjectorVersion).cross(CrossVersion.full)
-    ),
-    description                   := "Explore",
-    scalacOptions += "-Ymacro-annotations",
-    Global / onChangedBuildSource := ReloadOnSourceChanges,
-    scalafixDependencies ++= ClueGenerator.value ++ LucumaSchemas.value,
-    scalafixScalaBinaryVersion    := "2.13"
-  ) ++ lucumaPublishSettings
-)
+ThisBuild / description                := "Explore"
+ThisBuild / scalacOptions += "-Ymacro-annotations"
```

Comment:
Should this be aligned, maybe scalafmt it?

Reply:
Scalafmt is checked in CI, so yes it is formatted. It seems your current settings don't align these, probably because `:=` and `+=` are different.